### ISSUE TITLE
--no-show-signature breaks compatibility with git 2.7

### ISSUE THREAD

**Issue by Ccookkollars** [mtkennerly/dunamai#52]
Hi, I just encountered an issue using dunamai on centos. My understanding of the issue comes from reading about the issues others have encountered eg. https://github.com/conventional-changelog/conventional-changelog/issues/734. As a result, I'm not able to use poetry-dynamic-versioning.

```
>$ git --version
git version 2.8.3
>$ git log --no-show-signature
fatal: unrecognized argument: --no-show-signature
>$ poetry install

The command 'git log --no-show-signature -n 1 --pretty=format:"%cI"' returned code 128. Output:
fatal: unrecognized argument: --no-show-signature
```

Is the fix described here appropriate for this project? Or should the git required version be updated?
https://go-review.googlesource.com/c/vgo/+/127956/3/vendor/cmd/go/internal/modfetch/codehost/git.go

Thanks for your time!

**Comment by mtkennerly** [comment]
Thanks for reporting this. I can definitely make that change.

It would  be better if I could add Git 2.7.0 tests to CI, but I haven't managed to make that version build successfully :/

**Comment by mtkennerly** [comment]
This is now fixed in 1.14.1. You can use `poetry self add dunamai==1.14.1` to update.
